example/people/elsa.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: jac.nesto.ca/v1alpha1
+kind: Person
+metadata:
+  name: elsa
+spec:
+  firstName: Elsa
+  lastName: -
```

```suggestion
  lastName: smith
```
The lonely `-` here is causing the error you posted on slack

---

Ah! Thank you my friend.